src/Microsoft.AspNetCore.Identity/BuilderExtensions.cs

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-
-using Microsoft.AspNetCore.Identity;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
-
-namespace Microsoft.AspNetCore.Builder
-{
-    /// <summary>
-    /// Identity extensions for <see cref="IApplicationBuilder"/>.
-    /// </summary>
-    public static class BuilderExtensions
-    {
-        /// <summary>
-        /// Enables ASP.NET identity for the current application.
-        /// </summary>
-        /// <param name="app">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
-        /// <returns>The <see cref="IApplicationBuilder"/> instance this method extends.</returns>
-        public static IApplicationBuilder UseIdentity(this IApplicationBuilder app)
```

Comment:
Ah yeah good catch this PR was created before we decided to do that, I'll restore with obsolete